cmake/alpakaCommon.cmake

```diff
@@ -620,14 +617,13 @@ if(alpaka_ACC_SYCL_ENABLE)
         if(alpaka_SYCL_ONEAPI_GPU)
             # Create a drop-down list (in cmake-gui) of valid Intel GPU targets. On the command line the user can specifiy
             # additional targets, such as ranges: "Gen8-Gen12LP" or lists: "icllp;skl".
-            set(alpaka_SYCL_ONEAPI_GPU_DEVICES "bdw" CACHE STRING "Intel GPU devices / generations to compile for")
+            set(alpaka_SYCL_ONEAPI_GPU_DEVICES "intel_gpu_pvc" CACHE STRING "Intel GPU devices / generations to compile for")
             set_property(CACHE alpaka_SYCL_ONEAPI_GPU_DEVICES
-                        PROPERTY STRINGS "bdw;skl;kbl;cfl;bxt;glk;whl;aml;cml;icllp;lkf;ehl;tgllp;rkl;adl-s;adl-p;dg1;acm-g10;ats-m150;dg2-g10;acm-g11;ats-m75;dg2-g11;acm-g12;dg2-g12;pvc-sdv;pvc;gen11;gen12lp;gen8;gen9;xe;xe-hpc;xe-hpg")
+                        PROPERTY STRINGS "intel_gpu_pvc;intel_gpu_acm_g12;intel_gpu_acm_g11;intel_gpu_acm_g10;intel_gpu_dg1;intel_gpu_adl_n;intel_gpu_adl_p;intel_gpu_rpl_s;intel_gpu_adl_s;intel_gpu_rkl;intel_gpu_tgllp;intel_gpu_icllp;intel_gpu_cml;intel_gpu_aml;intel_gpu_whl;intel_gpu_glk;intel_gpu_apl;intel_gpu_cfl;intel_gpu_kbl;intel_gpu_skl;intel_gpu_bdw")
             # If the user has given us a list turn all ';' into ',' to pacify the Intel OpenCL compiler.
             string(REPLACE ";" "," alpaka_SYCL_ONEAPI_GPU_DEVICES "${alpaka_SYCL_ONEAPI_GPU_DEVICES}")
             
             target_compile_definitions(alpaka INTERFACE "ALPAKA_SYCL_ONEAPI_GPU")
-            target_link_options(alpaka INTERFACE "SHELL:-Xsycl-target-backend=${alpaka_SYCL_ONEAPI_GPU_TARGET} \"-device ${alpaka_SYCL_ONEAPI_GPU_DEVICES}\"")
```

Comment:
Why was this removed?

Reply:
It is implied by the new targets.

Reply:
I don't think this will work as expected. Or did Intel change the `-fsycl-targets` flag without documenting the change? https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2023-0/fsycl-targets.html

Setting `alpaka_SYCL_ONEAPI_GPU_TARGET` to `intel_gpu_pvc` instead of `spir64_gen` will likely produce an error.

Also you would need to set `alpaka_SYCL_ONEAPI_GPU_TARGET` somewhere below the GPU selection process, otherwise it will just be an empty string.

Reply:
Look [here](https://intel.github.io/llvm-docs/UsersManual.html#generic-options) for the `-fsycl-targets` option

Reply:
This is how I am testing the branch:
```bash
source /opt/intel/oneapi/setvars.sh --include-intel-llvm

cmake \
  -DCMAKE_CXX_COMPILER=/opt/intel/oneapi/compiler/latest/linux/bin/icpx \
  -DoneDPL_ROOT=/opt/intel/oneapi/dpl/latest \
  -DoneDPL_DIR=/opt/intel/oneapi/dpl/latest/lib/cmake/oneDPL \
  -DMKL_ROOT=/opt/intel/oneapi/mkl/latest \
  -DMKL_DIR=/opt/intel/oneapi/mkl/latest/lib/cmake/mkl \
  -DTBB_ROOT=/opt/intel/oneapi/tbb/latest \
  -DTBB_DIR=/opt/intel/oneapi/tbb/latest/lib/cmake/tbb \
  -DBUILD_TESTING=ON \
  -Dalpaka_ACC_SYCL_ENABLE=ON \
  -Dalpaka_SYCL_PLATFORM_ONEAPI=ON \
  -Dalpaka_SYCL_ONEAPI_GPU=ON \
  -Dalpaka_SYCL_ONEAPI_GPU_DEVICES=intel_gpu_acm_g10 \
  ../alpaka/

make -j`nproc`

make test
```

It works, and all tests pass:
```
100% tests passed, 0 tests failed out of 29

Total Test time (real) =  27.97 sec
```

Reply:
(to be honest that was before the latest rebase, I still have to test that)

Reply:
@AuroraPerego Oooh, nice. Is that already in the stable oneAPI compiler?

Reply:
Yes, it's in oneAPI 2023.2.0.

Reply:
Perfect :-)